data/seeds/02-recipes.js

```diff
@@ -0,0 +1,12 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('recipes').truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('recipes').insert([
+        {recipe_name: 'Deep Dish', dish_id: '1'},
+        {recipe_name: 'Club Sandwich', dish_id: '2'}
```

Comment:
It might be a little easier to generate recipe objects like this:
```javascript
return knex('recipes')
  .insert(['Deep Dish', 'Club Sandwich']
    .map((recipe_name, ind) => (
      { recipe_name, dish_id: `${ind + 1}` }
    ))
  );
```